### ISSUE TITLE
Hide/private posts or see deleted posts

### ISSUE THREAD

**Issue by JediMaster25** [LemmyNet/lemmy-ui#879]
There was a post I didn't want to keep public and deleted it, but then I wanted to see it again and couldn't. It would be nice to have a feature to hide or make our posts private, and a filter to see only those posts. Or otherwise be able to filter deleted posts which are already implemented.

**Comment by dessalines** [comment]
Hrm... this might be a bug. I'd thought that deleted posts were still accessible so they could be undeleted.

**Comment by JediMaster25** [comment]
Yes, I can see deleted posts, what I mean is a filter to find them. Since right now I would have to look for all the posts to find a deleted one.

**Comment by dessalines** [comment]
You mean from the search page? Right, we don't have that working right now, that's a lemmy backend issue. You can find them easily on the modlog page tho.

**Comment by JediMaster25** [comment]
Either from the search page or the user page, similar to the `Saved` filter.

**Comment by dessalines** [comment]
This is really a lemmy issue, because the back end can't do this yet.
